### ISSUE TITLE
Cannot fix params

### ISSUE THREAD

**Issue by MilanCugur** [jpmml/sklearn2pmml#236]
Sklearns **KNeighboursRegressor** params _weights='distance', metric='euclidean'_ cannot be fixed in **PMMLPipeline** if saving througs **sklearn2pmml.sklearn2pmml** is expected. 

Error: 

```
SEVERE: Failed to convert
java.lang.IllegalArgumentException: euclidean
	at sklearn.neighbors.KNeighborsUtil.encodeComparisonMeasure(KNeighborsUtil.java:160)
	at sklearn.neighbors.KNeighborsUtil.encodeNeighbors(KNeighborsUtil.java:112)
	at sklearn.neighbors.KNeighborsRegressor.encodeModel(KNeighborsRegressor.java:54)
	at sklearn.neighbors.KNeighborsRegressor.encodeModel(KNeighborsRegressor.java:29)
	at sklearn2pmml.pipeline.PMMLPipeline.encodePMML(PMMLPipeline.java:231)
	at org.jpmml.sklearn.Main.run(Main.java:233)
	at org.jpmml.sklearn.Main.main(Main.java:151)

Exception in thread "main" java.lang.IllegalArgumentException: euclidean
	at sklearn.neighbors.KNeighborsUtil.encodeComparisonMeasure(KNeighborsUtil.java:160)
	at sklearn.neighbors.KNeighborsUtil.encodeNeighbors(KNeighborsUtil.java:112)
	at sklearn.neighbors.KNeighborsRegressor.encodeModel(KNeighborsRegressor.java:54)
	at sklearn.neighbors.KNeighborsRegressor.encodeModel(KNeighborsRegressor.java:29)
	at sklearn2pmml.pipeline.PMMLPipeline.encodePMML(PMMLPipeline.java:231)
	at org.jpmml.sklearn.Main.run(Main.java:233)
	at org.jpmml.sklearn.Main.main(Main.java:151)
```

Occurs.
